CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## v2.0.0
  {release-date}
 ### Upgrade requirements
+- Only versions 1.23.2 and newer can be upgraded to this version. For older versions, please upgrade to last 1.23.x first.
```

Comment:
I prefer to update also documentation, namely data-storage.md, part GeoServer.

Reply:
Changed.